src/sentry/api/endpoints/event_file_committers.py

```diff
@@ -145,65 +32,28 @@ def get(self, request, project, event_id):
         # populate event data
         Event.objects.bind_nodes([event], 'data')
 
-        group = Group.objects.get(id=event.group_id)
-
-        first_release_version = group.get_first_release()
-
-        if not first_release_version:
-            return Response({'detail': 'Release not found'}, status=404)
-
-        releases = Release.get_closest_releases(project, first_release_version)
-
-        if not releases:
+        try:
+            committers = get_event_file_committers(
+                project,
+                event,
+                frame_limit=int(request.GET.get('frameLimit', 25)),
+            )
+        except Release.DoesNotExist:
             return Response({'detail': 'Release not found'}, status=404)
-
-        commits = self._get_commits(releases)
-
-        if not commits:
+        except Commit.DoesNotExist:
             return Response({'detail': 'No Commits found for Release'}, status=404)
 
-        frames = self._get_frame_paths(event)
-        frame_limit = int(request.GET.get('frameLimit', 25))
-        app_frames = [frame for frame in frames if frame['in_app']][-frame_limit:]
-        if not app_frames:
-            app_frames = [frame for frame in frames][-frame_limit:]
-
-        # TODO(maxbittker) return this set instead of annotated frames
-        path_set = {frame.get('filename') or frame['abs_path'] for frame in app_frames}
-
-        file_changes = []
-        if path_set:
-            file_changes = self._get_commit_file_changes(commits, path_set)
-
-        commit_path_matches = {
-            path: self._match_commits_path(file_changes, path) for path in path_set
-        }
-
-        annotated_frames = [
-            {
-                'frame': frame,
-                'commits': commit_path_matches[frame.get('filename') or frame['abs_path']]
-            } for frame in app_frames
-        ]
-
-        relevant_commits = list(
-            {commit for match in commit_path_matches for commit in commit_path_matches[match]}
-        )
-
-        committers = self._get_committers(annotated_frames, relevant_commits)
-        metrics.incr('feature.owners.has-committers', instance='hit' if committers else 'miss')
-
+        # XXX(dcramer): this data is unused, so lets not bother returning it for now
         # serialize the commit objects
-        serialized_annotated_frames = [
-            {
-                'frame': frame['frame'],
-                'commits': serialize(frame['commits'])
-            } for frame in annotated_frames
-        ]
+        # serialized_annotated_frames = [
+        #     {
+        #         'frame': frame['frame'],
+        #         'commits': serialize(frame['commits'])
+        #     } for frame in annotated_frames
+        # ]
 
         data = {
-            # map author ids to sentry user dicts
             'committers': committers,
-            'annotatedFrames': serialized_annotated_frames
+            # 'annotatedFrames': serialized_annotated_frames
```

Comment:
should this and the above just be deleted?

Reply:
I only kept it in that I think we're going to want it and its easier to uncomment code thatn it is to dig into git history